### PR TITLE
16650 button contrast issues

### PR DESCRIPTION
**Pull request by andrewgormley**
### Fixes: #16650

- [x]  (List page) Low contrast on buttons [both, multiple]
- [x]  (Racks) Low contrast download svg button [dark, multiple]
- [x]  (Buttons) Focus state on buttons is missing focus ring causing contrast and CVD issue [both, global]
- [x]  (Misc) Low contrast on cancel button [dark, multiple]
- [x]  (Misc) Low contrast on create combo button [dark, multiple]

### CODE REVIEW

**Comment by arthanson** [comment]
@andrewgormley just thought I'd mention - I actually find the black text on some of the buttons much harder to read, for example the export button with black on fuscia (see screenshot) sort of strains/hurts my eyes.  Also the delete button (black on red) isn't as bad but still seems less readable than before.  Not sure if it's just my eyes.

Devices | NetBox 2024-07-09 11-42-23
Devices | NetBox 2024-07-09 11-43-47
PP:B117 | NetBox 2024-07-09 11-44-46

**Comment by jsenecal** [comment]
I'm sorry if I come late to this discussion, but wouldn't it be best to address some of these color scheme issues as a user preference `Use high-contrast colors and buttons` ? 

This black on [green,orange,yellow,blue,purple] looks very harsh for the non-color impaired IMHO.

File: netbox/project-static/styles/overrides/_tabler.scss [hunk 1]
```diff
@@ -86,12 +133,17 @@ body[data-bs-theme=dark] {
   table {
     a {
       color: $bright-teal;
+
+      &.dropdown-item {
+        color: inherit;
+      }
```
Comment (jeremystretch) [inline netbox/project-static/styles/overrides/_tabler.scss:139]: Why is this needed?

File: netbox/project-static/styles/overrides/_tabler.scss [hunk 2]
```diff
@@ -70,6 +76,47 @@ table th.orderable a {
...
+
+// Fix secondary combo button contrast
+.btn-outline-secondary {
+  &.active {
+    color: var(--tblr-primary-fg);
+  }
+  &:hover {
+    color: var(--tblr-primary-fg);
+  }
+}
```
Comment (jeremystretch) [inline netbox/project-static/styles/overrides/_tabler.scss:93]: This doesn't seem to have any effect? A button with `btn-outline-secondary` appears to render the same whether or not this rule is in place AFAICT.

**Review by jeremystretch — CHANGES_REQUESTED** [review]

**Comment by jeremystretch** [comment]
I realize this is subjective, but I'm not a fan of the brighter colors. As other have noted, it makes the text more difficult to read IMO.

Original:
screenshot1

Modified:
screenshot2

**Review by jeremystretch — APPROVED** [review]

**Comment by jeremystretch** [comment]
I've attached the [reverted changes](https://github.com/user-attachments/files/16255067/pr16845_reverted.txt) for future reference.

**Pull request merged**